source/aggregation-pipeline-builder.txt

```diff
@@ -231,10 +231,9 @@ To export results from your aggregation pipeline:
 
    Under :guilabel:`Select Export File Type`, select either 
    :guilabel:`JSON` or :guilabel:`CSV`. If you select :guilabel:`JSON`,
-   your data is exported to the target file as a comma-separated 
-   array of JSON objects.
+   your data is exported to the target file as an array of JSON objects.
 
-   Then, under :guilabel:`Output`, choose where to export the file to.
+#. Then, under :guilabel:`Output`, choose where to export the file to.
```

Comment:
Now that this is it's own step, I think we can omit "Then, " and just make this:

```
Under :guilabel:`Output`, choose where to export the file to.
```

Reply:
Done.